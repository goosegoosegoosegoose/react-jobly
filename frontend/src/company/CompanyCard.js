import React from "react";

const CompanyCard = ({handle, name, description, logoUrl}) => {
  return(
    <a href={`/companies/${handle}`}>
      <h5>{name}</h5>
      <p>{description}</p>
      <img src={logoUrl} alt={logoUrl} />
    </a>
  )
}

export default CompanyCard;