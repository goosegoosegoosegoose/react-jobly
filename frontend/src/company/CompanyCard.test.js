import { render } from '@testing-library/react';
import CompanyCard from './CompanyCard';

test('smoke test', () => {
  render(      
    <CompanyCard />
  );
});

test('snapshot', () => {
  const { asFragment } = render(<CompanyCard />);
  expect(asFragment()).toMatchSnapshot();
});