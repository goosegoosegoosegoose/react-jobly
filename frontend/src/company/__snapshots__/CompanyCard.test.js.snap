// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <a
    href="/companies/undefined"
  >
    <h5 />
    <p />
    <img />
  </a>
</DocumentFragment>
`;
